apis/notifications.md

```diff
@@ -129,6 +129,71 @@ Build and run your app again, and press the "Notify" button. Notice that the not
 Remember that `SimpleAction`s added in the `Application` class with `add_action ()` are automatically added in the `app` namespace. Notifications can't trigger actions defined in other namespaces like `win`.
 {% endhint %}
 
+## Default Action
+
+You may have noticed that when you click on a new notification, a new window pops up. This is happening because the notification has a default action that is executed when the user clicks on it. If you don't set it, it will `activate` your application, where we create a new window and present it.
+
+You can change the default action using  [`set_default_action ()`](https://valadoc.org/gio-2.0/GLib.Notification.set\_default\_action.html)
```

Comment:
It's probably worth mentioning that notification actions have to be in the `app` namespace. Maybe link back to the section on Actions?